edu.cuny.hunter.streamrefactoring.core/src/edu/cuny/hunter/streamrefactoring/core/analysis/Util.java

```diff
@@ -458,7 +488,8 @@ static int getLineNumberFromIR(IBytecodeMethod method, SSAInstruction instructio
 					// type is java.lang.Object.
 					// Find the return type of the instruction.
 					TypeInference inference = TypeInference.make(node.getIR(), false);
-					Collection<TypeAbstraction> returnTypes = Util.getPossibleTypes(valueNumber, inference);
+					// Get all possible types
+					Collection<TypeAbstraction> returnTypes = Util.getAllPossibleTypes(valueNumber, inference);
```

Comment:
Using `getPossibleTypes()` is not safe. So I modified it to `getAllPossibleTypes()`